client/src/components/Main/EventCard/index.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import styled from "styled-components";
+import {theme1, theme2} from "../../../resources/style-constants"
+import ActionCard from "../ActionCard"
+
+const CardStyle = styled.div`
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+`;
+
+const TextBar = styled.div`
+font-size: 1.8vw;
+display: flex;
+flex-direction: row;
+justify-content: center;
+align-items: center;
+position: absolute;
+left: 0;
+bottom: 0;
+`;
+
+const ColorBar = styled.div`
+background-color: ${({ color }) => color};
+opacity: .9;
+border-top-left-radius: 7px;
```

Comment:
You should use borderRadius from the style-constants here and the other places where you set the border-radius for consistency.